src/main/java/io/vertx/codegen/CodeGen.java

```diff
@@ -193,18 +176,19 @@ private Resolved resolveMember(Element elt, TypeMirror type, String member) {
     throw new GenException(elt, "Only declared element are supported");
   }
 
-  private void processSerializerOrDeserializer(TypeElement converterElt, /*TypeElement typeElt, */List<String> selectors, ExecutableType methodType) {
+  private void processConverter(TypeElement converterElt, TypeMirror type, List<String> selectors, ExecutableType methodType) {
```

Comment:
Can you be more specific about what is `type`? It took me a while to understand it, maybe `targetType` or `targetDataObjectType`?

Reply:
I will change this